package.json

```diff
@@ -100,7 +100,7 @@
     "npm": "~8"
   },
   "lint-staged": {
-    "*": "prettier --ignore-unknown --write",
+    "!(./jslib/**)*": "prettier --ignore-unknown --write",
```

Comment:
Do we need to specify `/**` here?

Reply:
Apparently not, I removed it and it still tested fine.